rust/stackable-cockpit/src/utils/k8s/client.rs

```diff
@@ -280,15 +291,15 @@ impl KubeClient {
     /// about allocatable resources. These values don't reflect currently
     /// available resources.
     pub async fn get_cluster_info(&self) -> Result<ClusterInfo> {
-        let node_api: Api<Node> = Api::all(self.client.clone());
-        let nodes = node_api
-            .list(&ListParams::default())
-            .await
-            .context(KubeSnafu)?;
-
+        let nodes = self.list_nodes().await?;
         ClusterInfo::from_nodes(nodes).context(ClusterSnafu)
     }
 
+    pub async fn get_endpoint(&self, namespace: &str, name: &str) -> Result<Endpoints> {
```

Comment:
I guess we should rename the function:

```suggestion
    pub async fn get_endpoints(&self, namespace: &str, name: &str) -> Result<Endpoints> {
```

Reply:
My fault. It's a single k8s object, but itself is already called `Endpoints` :D
Fixed in b961dabd6364610b7df1ed295fe7e8c037efb5b1